evalml/pipelines/components/ensemble/stacked_ensemble_base.py

```diff
@@ -82,29 +80,3 @@ def default_parameters(cls):
                 'cv': None,
                 'n_jobs': 1,
                 }
-
-    def fit(self, X, y=None):
```

Comment:
Realized this was unnecessary since it inherits from the `Estimator` class. Deleting!